crates/hdk/tests/tests/hdk_extern/hdk_extern_invalid_return.stderr

```diff
@@ -0,0 +1,8 @@
+error: functions marked with #[hdk_extern] must return `ExternResult<String>`
```

Comment:
This feels vaguely misleading. I wonder if rephrasing slightly to `This function is marked #[hdk_extern] so it must return `ExternResult<String>` instead of `String`.

Otherwise it sounds like all zome functions must return a string

Reply:
I see, improved the naming